orchestration/_tests/test_globus_flow.py

```diff
@@ -51,6 +51,123 @@ def __init__(self, root_path, uuid_value=None):
+            "nersc733_alsdev_raw": MockEndpoint(root_path="mock_nersc733_alsdev_raw_path", uuid_value=str(uuid4())),
+        }
+
+        # Define mock apps
+        self.apps = {
+            "als_transfer": "mock_als_transfer_app"
+        }
+
+        # Use the mock transfer client instead of the real TransferClient
+        self.tc = MockTransferClient()
+
+        # Set attributes for easy access
+        self.data733_raw = self.endpoints["data733_raw"]
+        self.nersc733_alsdev_raw = self.endpoints["nersc733_alsdev_raw"]
+
+
+def test_process_new_733_file(mocker: MockFixture) -> None:
+    """
+    Test the process_new_733_file flow from orchestration.flows.bl733.move.
+
+    This test verifies that:
+      - The get_transfer_controller function is called (patched) with the correct parameters.
+      - The returned transfer controller's copy method is called with the expected file path,
+        source, and destination endpoints from the provided configuration.
+
+    Parameters:
+        mocker (MockFixture): The pytest-mock fixture for patching and mocking objects.
+    """
+    # Import the flow to test.
+    from orchestration.flows.bl733.move import process_new_733_file
+
+    # Patch the Secret.load and init_transfer_client in the configuration context.
+    with mocker.patch('prefect.blocks.system.Secret.load', return_value=MockSecret()):
+        mocker.patch(
+            "orchestration.flows.bl733.config.transfer.init_transfer_client",
+            return_value=mocker.MagicMock()  # Return a dummy TransferClient
+        )
+        from orchestration.flows.bl733.config import Config733
+
+    # Patch the schedule_prefect_flow call to avoid real Prefect interaction
+    mocker.patch(
+        "orchestration.flows.bl733.move.schedule_prefect_flow",
+        return_value=None
+    )
+
+    # Instantiate the dummy configuration.
+    mock_config = Config733()
+
+    # Generate a test file path.
+    test_file_path = f"/tmp/test_file_{uuid4()}.txt"
+
+    # Create a mock transfer controller with a mocked 'copy' method.
+    mock_transfer_controller = mocker.MagicMock()
+    mock_transfer_controller.copy.return_value = True
+
+    # Patch get_transfer_controller where it is used in process_new_733_file.
+    mocker.patch(
+        "orchestration.flows.bl733.move.get_transfer_controller",
+        return_value=mock_transfer_controller
+    )
+
+    # Execute the flow with the test file path and dummy configuration.
+    result = process_new_733_file(file_path=test_file_path, config=mock_config)
+
+    # Verify that the transfer controller's copy method was called exactly once.
+    assert mock_transfer_controller.copy.call_count == 1, "Transfer controller copy method should be called exactly once"
+    assert result is None, "The flow should return None"
+
+
+# ----------------------------
+# Tests for 832
+# ----------------------------
```

Comment:
I think it would be nice to move the tests for each beamline into their own folder with this directory structure.

```
orchestration/
├── flows/
│   ├── bl733/
│   │   ├── __init__.py
│   │   ├── move.py
│   └── bl832/
│       ├── __init__.py
│   │   ├── move.py
└── tests/
    ├── test_bl733/
    │   ├── __init__.py
    │   ├── test_move.py
    └── test_bl832/
        ├── __init__.py
        └── test_move.py
```

That way we can also put your `MockGlobusComputeClient` in a shared location at the test level or a in a shared pytest fixture, and you can run tests for each beamline separately more easily. Would that work? What do you think?

I don't think it needs to be part of this PR though. Just a follow up after this is merged.